docs/public/installing-odo.adoc

```diff
@@ -46,28 +46,28 @@ The following section describes how to install `odo` on different platforms via
 
 === Binary installation
 
-. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo.exe`] file.
-. Add the location of your `odo.exe` to your `GOPATH/bin` directory.
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo-windows-amd64.exe`] file.
+. Rename the downloaded file to `odo.exe`
+. Add the location of your `odo.exe` to your `GOPATH/bin` directory. For details on how to set `GOPATH` see link:https://golang.org/doc/gopath_code.html#GOPATH[`The GOPATH environment variable`]
```

Comment:
Shouldn't we make this more generic? 
I was hoping that we could write this guide in a way that wouldn't include any Go specific things like `GOPATH`.
Odo users might not be familiar with Go and using terms like `GOPATH` might confuse them.

Reply:
agreed odo doesn't need to moved to GOPATH/bin - just add it to any folder and add that folder path to the` %PATH% `

Reply:
we have used `C:\Go\bin` below to show how to add it to PATH so maybe use that as an example folder here as well